test_saas/integration/test_path.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+from typing import ByteString
+import pytest
+import exasol.bucketfs as bfs
+
+
+@pytest.fixture
+def children_poem() -> ByteString:
+    poem_text = \
+        b"Twinkle twinkle little star." \
+        b"How I wonder what you are." \
+        b"Up above the world so high." \
+        b"Like a diamond in the sky."
```

Comment:
```suggestion
    poem_text = (
        b"Twinkle twinkle little star."
        b"How I wonder what you are."
        b"Up above the world so high."
        b"Like a diamond in the sky."
     )
```

Reply:
Using `()` to span an expression across multiple lines usually is easier to maintain and update.